modules/programs/comodoro.nix

```
@@ -0,0 +1,91 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (config.programs) comodoro;
+  tomlFormat = pkgs.formats.toml { };
+
+  server = config.services.comodoro;
+  serverArgs = with server.settings; {
+    inherit preset;
+    protocols = if lib.isList protocols then
+      lib.concatStringsSep " " protocols
+    else
+      protocols;
+  };
+
+in {
+  meta.maintainers = with lib.hm.maintainers; [ soywod ];
+
+  options = {
+    programs.comodoro = {
+      enable =
+        lib.mkEnableOption "Enable Comodoro, the CLI to manage your time.";
```

`mkEnableOption` automatically adds "Enable " to the front and "." to the end.

```suggestion
        lib.mkEnableOption "Comodoro, the CLI to manage your time";
```